source/install/self-managed-changelog.md

```diff
@@ -6,11 +6,201 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
+
+**IMPORTANT:** If you upgrade from a release earlier than v7.10, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Highlights
+
+#### Persistent Notifications
+ - Added a persistent notification option when sending urgent priority posts.
+
+#### Apps Bar
+ - The apps bar is now enabled by default for on-prem servers. ``ExperimentalSettings.EnableAppBar`` was also renamed to ``ExperimentalSettings.DisableAppBar``. See more details at:
+   - https://docs.mattermost.com/configure/experimental-configuration-settings.html#disable-app-bar 
+   - https://forum.mattermost.com/t/channel-header-plugin-changes/13551
+
+#### Calls
+ - 
+
+#### Playbooks
+ - 
+
+#### Boards
+ - Prepackaged Boards version 7.11.0.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a **Mattermost Marketplace** option to the bottom of the apps bar. The option is visible when the Marketplace is enabled, and the user has ``SYSCONSOLE_WRITE_PLUGINS`` permissions.
+ - Added an **Add channels** button to the bottom of the left-hand sidebar to make the action more obvious for users who want to create or join channels.
+ - Removed the Webapp Build Hash from **Main Menu > About Mattermost** since it is now identical to Server Build Hash.
+ - Replaced the ``compass-components`` icon component with ``compass-icons``.
+ - Added “hours ahead” timezone details to the user profile popover.
+ - Added an experimental feature to disable re-fetching of channel and channel members on browser focus.
+ - Bot users are now hidden in the user selector in apps forms.
+ - Removed the fetching of archived channels on page load.
+ - The **Channel Type** dropdown within the **Browse Channels** modal can now be focused.
+ - Removed in-app help pages that were no longer accessible.
+ - Removed system join/leave messages from thread replies and post them instead in the main channel.
+ - Added an experimental setting to make the channel autocomplete only appear after typing two characters instead of immediately after the tilde (~).
+ - Users with default profile pictures will now regenerate a new picture when their username is changed.
+ - Implemented URL auto generation on channel creation for when there's no URL safe characters on its name.
+ - Added a new option to auto-follow all threads in the channel **Notification Preference** settings.
+ - ``CTRL/CMD + K`` shortcut can now be used to insert link formatting when text is selected.
+ - ``pas`` and ``pascal`` code blocks are now higlighted.
+ - Language setting in Boards was removed. The main language setting under **Settings -> Display Settings** now covers Boards. Boards previously supported Catalan, Greek, Indonesian, and Occitan, but since these 4 languages were only partially translated (<40%; Boards-only), they have been removed until more areas of Mattermost are translated into these languages.
```

Comment:
Propose remove from changelog since it's related to the focalboard plugin itself.

I propose these types of plugin changes go in the v7.11 focalboard changelog rather than the server changelog

Reply:
@sbishel Is this and the below notes from Eric already in the Focalboard changelog?